security/xca/PortFile

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                  1.0
+
+name                        xca
+version                     2.1.2
+revision                    0
+
+categories                  security crypto
+platforms                   darwin
+license                     xca
+
+maintainers                 {@lhaeger} openmaintainer
+
+description                 X Certificate and Key management
+long_description            XCA is a CA & PKI management tool that supports creating and managing X.509 certificates, \
+                            certificate requests, RSA, DSA and EC private keys, Smartcards and CRLs.
+homepage                    https://www.hohnstaedt.de/xca/
+
+master_sites                https://github.com/chris2511/xca/releases/download/RELEASE.${version}/
+distname                    ${name}-${version}
```

Comment:
Please instead use the GitHub portgroup which will simplify the portfile because `master_sites` and `distname` will not be needed because the portgroup will set them properly.

Reply:
Great suggestion, another MacPorts feature I did not know about yet... thanks!